scapy/arch/windows/__init__.py

```diff
@@ -397,7 +397,7 @@ def read_routes():
         elif release == "XP":
             routes = read_routes_xp()
         else:
-            routes = read_routes_7()
+            routes = read_routes_post2008()
```

Comment:
This was to test the function. That's why it shouldn't be merged.... read_routes_post2008 is correctly working, so i don't know why it was marked as "TODO"